pkg/build/golang/docker.go

```diff
@@ -62,8 +63,17 @@ func (b *DockerGoBuilder) Build(in *api.BuildInput) (*api.BuildOutput, error) {
 	// TODO support specifying a docker endpoint + TLS parameters from env.
 	// raulk: I don't see a need for this now, as we certainly want to do builds
 	// locally, and push the image to a registry.
+	// aschmahmann: No host config parameters means we need some OS check
+
+	var host string
+	if runtime.GOOS == "windows" {
```

Comment:
We should be able to replace this with `client.DefaultDockerHost`. The docker project uses OS build constraints (see https://github.com/moby/moby/blob/master/client/client_windows.go and https://github.com/moby/moby/blob/master/client/client_unix.go) to vary the value based on the target OS.

Reply:
even better!

Reply:
@aschmahmann can you test this?

Reply:
@daviddias If we're using the `client.DefaultDockerHost` it should be fine.